lib/motsu-proc/Cargo.toml

```diff
@@ -2,7 +2,6 @@
 name = "motsu-proc"
 description = "Mostu's Procedural Macros"
 edition.workspace = true
```

Comment:
This will be published right? Shouldn't we add keywords as well?

Reply:
But it cost nothing to duplicate keywords here also. I'll update them